src/components/WalletSection.tsx

```diff
@@ -1,29 +1,22 @@
-import PropTypes from 'prop-types';
-import React from 'react';
+import React, {ReactNode} from 'react';
 import useThemeStyles from '@styles/useThemeStyles';
 import Section from './Section';
 
-const propTypes = {
+type WalletSectionProps = {
     /** Contents to display inside the section */
-    children: PropTypes.node,
+    children?: ReactNode;
 
     /** The icon to display along with the title */
-    icon: PropTypes.func,
+    icon?: (props: unknown) => React.ReactNode;
```

Comment:
```suggestion
    icon?: React.FC<SvgProps>;
```

Reply:
wondering if we shouldn't use here Icon type from Onyx types? @VickyStash

Reply:
@kubabutkiewicz I've checked the `WalletSection` component usage, and there just svg icon provided for the icon param